### ISSUE TITLE
Normalization operations

### ISSUE THREAD

**Issue by jjsjann123** [csarofeen/pytorch#515]
Issues regarding normalization operations that should be tracked:

1. autodiff is disabled for batch_norm & layer_norm (https://github.com/pytorch/pytorch/issues/19769)

2. `aten::batch_norm` schema doesn't mark in-place update for running_mean & running_var. This is a hot potato, as simply marking them as in-place could impact fusion pretty seriously due to the lack of accurate alias analysis.

3. Extending from (2), in-place update of running stats require codegen support for that.

4. Does normalization scheduling require specific partition logic? This part is still a blank in our implementation. We could start with at least marking normalization operations as a new op in parser.

**Comment by jjsjann123** [comment]
Tagging #440, I need some update on what patterns to be expected for normalization scheduling in order to support it properly in integration.

**Comment by csarofeen** [comment]
I think these issues have been resolved, can we close this? @jjsjann123

**Comment by jjsjann123** [comment]
Normalization issues marked here has been properly tackled. closing.
